.conan-recipes/llnl-units/conanfile.py

```diff
@@ -0,0 +1,79 @@
+from conans import ConanFile, CMake, tools
+import os
+
+CMAKE_PROJECT_STR = """project(
+    UNITS
+    LANGUAGES CXX
+    VERSION 0.5.0
+)"""
+
+
+class UnitsConan(ConanFile):
+    name = "LLNL-Units"
+    version = "0.5.0"
```

Comment:
We need a unique version string for each change to the repo. And we should also specify a revision of the repo so that we fix a specific version of the source and not always pull `main`.

Maybe this is an option, at least once it is stabilised: https://docs.conan.io/en/latest/reference/conanfile/attributes.html#scm